frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts

```diff
@@ -157,7 +157,10 @@ export const infiniteListLogic = kea<infiniteListLogicType>({
                         ),
                         searchQuery: values.searchQuery,
                         queryChanged,
-                        count: response.count || 0,
+                        count:
+                            response.count ||
+                            (Array.isArray(response) ? response.length : 0) ||
```

Comment:
When does `response.count` not cover the result we need? Just for paths results?

Reply:
for paths, and everything that's about to come with universal search, those have different response structures too.

Reply:
https://github.com/PostHog/posthog/pull/9299/files#diff-b25cfa2d4b96abd40174cb762dab262471bcb6216e33e86d49c9d612d963cc67R173